sematic/ui/src/components/LogPanel.tsx

```diff
@@ -1,5 +1,5 @@
 import { Alert, Box, TextField } from "@mui/material";
```

Comment:
Where is the `Filter` form rendered now?
Could you please add a screenshot with the zoomed-out complete page view?

Reply:
`Filter` form is not rendered in case of exceptions. (The enclosing `<Box>` is hidden. The `Filter` is the `<TextBox>` component)

Here is the complete page view:
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/1046489/208506096-204d0803-1fd1-4494-9c8f-6c3c12c37293.png">